manifests/minio-operator-rhmp.clusterserviceversion.yaml

```diff
@@ -685,7 +685,7 @@ spec:
                     env:
                       - name: CONSOLE_OPERATOR_MODE
                         value: "on"
-                    image: quay.io/minio/console@sha256:dd181fd701b6a7e7787d7f5b63653367aca2bcac151a5c204152a44a7604c092
+                    image: quay.io/minio/console@
```

Comment:
is this okay? @dvaldivia

Reply:
@cniackz this doesn't seem right, right?

Reply:
fixed @harshavardhana